### DISCUSSION TITLE
Question about documentation tools

### DISCUSSION THREAD

**Discussion by itague** [jongpie/NebulaLogger#435] [Q&A] [answered]
I know that I could be off topic, but I'm very curious to know which tool you are using to diagram the Nebula logger architecture and to write down all project documentation.

Thanks in advance for your answer.

**Comment by jongpie** [comment, marked as answer]
Hi @itague! Sure, I'm happy to share the tools that I'm currently using:

- Diagrams: I use [LucidChart](https://www.lucidchart.com/) for all diagrams. LucidChart provides some [Salesforce architecture diagram templates](https://www.lucidchart.com/blog/salesforce-shape-library) that I really like to use, I think the provided shapes look really nice.
- [GitHub Pages site](https://jongpie.github.io/NebulaLogger/): this site is automatically generated based on code-comments that I add in Apex & JavaScript (LWC). Code-comments are automatically converted to markdown files that live in the `docs` folder](https://github.com/jongpie/NebulaLogger/tree/v4.9.9/docs)
  - [ApexDoc markdown files](https://github.com/jongpie/NebulaLogger/tree/v4.9.9/docs/apex) are generated using [apexdocs](https://github.com/cesarParra/apexdocs) that's maintained by @cesarParra
  - [JavaScript markdown files for lightning components](https://github.com/jongpie/NebulaLogger/tree/v4.9.9/docs/lightning-components) are generted using [JSDoc] (https://github.com/jsdoc/jsdoc) 

For all of the other content in [`README.md`](https://github.com/jongpie/NebulaLogger/tree/v4.9.9/README.md) and [the wiki](https://github.com/jongpie/NebulaLogger/wiki), I just manually write/edit the content in VS Code (using some markdown plugins).

Hope this helps! Let me know if you have any follow up questions.
